docs/website/content/v0.3/en/configuration/v1alpha1.md

```diff
@@ -0,0 +1,276 @@
+---
+title: v1alpha1
+---
+
+Package v1alpha1 represents a machine configuration file.
+
+## Generating the Base Machine Configuration Files
+
+We can generate the files using `osctl`.
+This configuration is enough to get started in most cases, however it can be customized as needed.
+
+```bash
+osctl config generate --version v1alpha1 <cluster name> <cluster endpoint>
+````
+
+This will generate a machine config for each node type, and a talosconfig.
+The following is an example of an `init.yaml`:
+
+```yaml
+version: v1alpha1
+machine:
+  type: init
+  token: 5dt69c.npg6duv71zwqhzbg
+  ca:
+    crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUJIekNCMHFBREFnRUNBaEIwL2grQ2hocGhWUXhXQ2x4dSsxSjVNQVVHQXl0bGNEQVFNUTR3REFZRFZRUUsKRXdWMFlXeHZjekFlRncweE9URXhNRGt4T0RRM016bGFGdzB5T1RFeE1EWXhPRFEzTXpsYU1CQXhEakFNQmdOVgpCQW9UQlhSaGJHOXpNQ293QlFZREsyVndBeUVBWXRKQ2FkWnhOK1A2eUVlNHNEM2dLQXJ2Y1ZmU3JqdlJ5YzNmCnRhTHorcFNqUWpCQU1BNEdBMVVkRHdFQi93UUVBd0lDaERBZEJnTlZIU1VFRmpBVUJnZ3JCZ0VGQlFjREFRWUkKS3dZQkJRVUhBd0l3RHdZRFZSMFRBUUgvQkFVd0F3RUIvekFGQmdNclpYQURRUUNrczNuMUFNeDhBcGluaEk1YgpaQU9IVFJ0dEZiSXQ3RmU1VFJ2aEdvN21rR0FnWmI5M21ZbGx4NFZjL2VKc0p0aVJURHN6cVp5VnlUVWVCaU45CnF5b0wKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
i feel like we should try to mask or abbreviate the certificates here. It's not super helpful to see these giant blocks.